layouts/shortcodes/observability_pipelines/configure_existing_pipelines/source_env_vars/splunk_hec.md

```diff
@@ -0,0 +1,4 @@
+- Splunk HEC address:
+    - The bind address that your Observability Pipelines Worker listens on to receive logs originally intended for the Splunk indexer. For example, `0.0.0.0:8088`   
+    **Note**: `/services/collector/event` is automatically appended to the endpoint.
+    - Stored in the environment variable:`DD_OP_SOURCE_DATADOG_AGENT_ADDRESS` `DD_OP_SOURCE_SPLUNK_HEC_ADDRESS`.
```

Comment:
```suggestion
    - Stored in the environment variables`DD_OP_SOURCE_DATADOG_AGENT_ADDRESS` and `DD_OP_SOURCE_SPLUNK_HEC_ADDRESS`.
```

Reply:
Also edited the suggestion to remove the DD env var, it doesn't belong here